src/bionics.cpp

```diff
@@ -150,174 +150,178 @@ bool player::activate_bionic( int b, bool eff_only )
+        if( has_effect( effect_pblue ) ) {
+            good.push_back( _( "Prussian Blue" ) );
         }
-        if (has_effect( effect_iodine )) {
-            good.push_back(_("Potassium Iodide"));
+        if( has_effect( effect_iodine ) ) {
+            good.push_back( _( "Potassium Iodide" ) );
         }
-        if (has_effect( effect_datura )) {
-            good.push_back(_("Anticholinergic Tropane Alkaloids"));
+        if( has_effect( effect_datura ) ) {
+            good.push_back( _( "Anticholinergic Tropane Alkaloids" ) );
         }
-        if (has_effect( effect_took_xanax )) {
-            good.push_back(_("Xanax"));
+        if( has_effect( effect_took_xanax ) ) {
+            good.push_back( _( "Xanax" ) );
         }
-        if (has_effect( effect_took_prozac )) {
-            good.push_back(_("Prozac"));
+        if( has_effect( effect_took_prozac ) ) {
+            good.push_back( _( "Prozac" ) );
         }
-        if (has_effect( effect_took_flumed )) {
-            good.push_back(_("Antihistamines"));
+        if( has_effect( effect_took_flumed ) ) {
+            good.push_back( _( "Antihistamines" ) );
         }
-        if (has_effect( effect_adrenaline )) {
-            good.push_back(_("Adrenaline Spike"));
+        if( has_effect( effect_adrenaline ) ) {
+            good.push_back( _( "Adrenaline Spike" ) );
         }
-        if (has_effect( effect_adrenaline_mycus )) {
-            good.push_back(_("Mycal Spike"));
+        if( has_effect( effect_adrenaline_mycus ) ) {
+            good.push_back( _( "Mycal Spike" ) );
         }
-        if (has_effect( effect_tapeworm )) {  // This little guy is immune to the blood filter though, as he lives in your bowels.
-            good.push_back(_("Intestinal Parasite"));
+        if( has_effect(
+                effect_tapeworm ) ) { // This little guy is immune to the blood filter though, as he lives in your bowels.
+            good.push_back( _( "Intestinal Parasite" ) );
         }
-        if (has_effect( effect_bloodworms )) {
-            good.push_back(_("Hemolytic Parasites"));
+        if( has_effect( effect_bloodworms ) ) {
+            good.push_back( _( "Hemolytic Parasites" ) );
         }
-        if (has_effect( effect_brainworms )) {  // These little guys are immune to the blood filter too, as they live in your brain.
-            good.push_back(_("Intracranial Parasite"));
+        if( has_effect(
+                effect_brainworms ) ) { // These little guys are immune to the blood filter too, as they live in your brain.
+            good.push_back( _( "Intracranial Parasite" ) );
         }
-        if (has_effect( effect_paincysts )) {  // These little guys are immune to the blood filter too, as they live in your muscles.
-            good.push_back(_("Intramuscular Parasites"));
+        if( has_effect(
+                effect_paincysts ) ) { // These little guys are immune to the blood filter too, as they live in your muscles.
```

Comment:
These changes are *really* ugly. It would be much better to move the comment on a separate line (inside the branch?) and keep the while `if(...) {` part on one line.